scripts/start-ganache.sh

```diff
@@ -1 +1 @@
-./node_modules/ganache-cli/build/cli.node.js --defaultBalanceEther 1000000 --deterministic --a 15 > ganache-blockchain-log.txt &
\ No newline at end of file
+./node_modules/ganache-cli/cli.js --defaultBalanceEther 1000000 --deterministic --a 15 > ganache-blockchain-log.txt &
```

Comment:
Note:

source of this change is 
https://github.com/centrehq/centre-tokens/commit/1503358c29490677fcf0be9429e1bb245c47be98#diff- s776a1b47d4896c61f45190bd3ba2c746582fdbef396eb83caad709cf84ff5205L1

The script isn't used in code in centre-tokens, so I'd be fine with deleting it entirely. But, if we're keeping the file, we should make this change.

Reply:
if we're not using it, we might as well delete it

Reply:
sounds good, deleted https://github.com/centrehq/centre-tokens/pull/327/commits/572fd5f034feee44f88ff6f8c7d5fb45e5d55756